### PR TITLE
MEPTS-1409 | TXTB CHANGES MARCH RELEASE

### PR DESCRIPTION
**Pull request by helderjosue**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERSemiAnnualReport.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public List<ReportDesign> constructReportDesigns(ReportDefinition reportDefiniti
       reportDesign =
           createXlsReportDesign(
               reportDefinition,
-              "Template_PEPFAR_MER_2.6.1_Semiannual_Report_v1.5.2.xls",
+              "Template_TX TB_PEPFAR_MER_2.7_Semiannual_Report_v.0.3.xls",
```
Comment (ItsMurumba) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERSemiAnnualReport.java:91]: Is there a reason we are downgrading the version of the template from v1.5.3 to v0.3? @helderjosue

**Review by ItsMurumba — COMMENTED** [review]
Kindly address the comment raised.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERSemiAnnualReport.java (hunk 1 above)
Comment (helderjosue) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERSemiAnnualReport.java:91]: The version was upgraded from **MER_2.6.1_Semiannual** to **MER_2.7_Semiannual**
And its actually a temporary template that will be used for txtb, the BA team is going to provide the final template once its approved @ItsMurumba

Comment (ItsMurumba) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERSemiAnnualReport.java:91]: I see, I had not seen the 2.7 part. Thank you for the clarification @helderjosue

**Review by ItsMurumba — APPROVED** [review]
LGTM

**Pull request merged**